### ISSUE TITLE
Python 3 issue

### ISSUE THREAD

**Issue by falematte** [augustt198/latex2sympy#47]
How to work with python 3?

**Comment by amalizzio** [comment]
Yes, please, I tried and I get issues with importing the gen.PSParse

**Comment by frucci** [comment]
python3!!

> Yes, please, I tried and I get issues with importing the gen.PSParse

**Comment by westurner** [comment]
- https://pypi.org/project/latex2sympy3
  - https://github.com/jackatbancast/latex2sympy
    - https://github.com/augustt198/latex2sympy/compare/master...jackatbancast:master

**Comment by westurner** [comment]
Does the new `sympy.parsing.latex.parse_latex()` work in Python 3?

- https://github.com/sympy/sympy/pull/13706
- https://github.com/augustt198/latex2sympy/issues/11

**Comment by westurner** [comment]
> Yep, it works in python 2 and 3.

https://github.com/sympy/sympy/pull/13706#issuecomment-460724996

Thank you for this!
